### ISSUE TITLE
[Update request] Bump to QT6.

### ISSUE THREAD

**Issue by Hexality** [Houston4444/Patchance#9]
many distros are taking qt5 out of their repository, would be interesting if you could update the defaults of the application too qt6.

**Comment by Houston4444** [comment]
> would be interesting if you could update the defaults of the application too qt6.

Of course, one day in this year I will do it. I don't know if it can be done quickly or not.

 > many distros are taking qt5 out of their repository,

Sorry to say that, but, remove qt5 from the repos while KDE Plasma 6 has not been released, it is just a stupid choice.

**Comment by Houston4444** [comment]
Hi, It is now possible to use Patchance with PyQt6. Feedback is welcome of course.
